src/seedu/addressbook/data/person/Person.java

```diff
@@ -73,6 +73,16 @@ public boolean equals(Object other) {
                 && this.hasSameData((ReadOnlyPerson) other));
     }
 
+    @Override
+    public int compareTo(ReadOnlyPerson other) {
+
+        if(other == this || other == null){ // return 0 if same object or null
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) violation! One more more of these problems detected:
1. extra/missing spaces.
2. brace style problem.
3. missing braces.
4. name doesn't follow naming convention